nautobot/extras/management/commands/remove_stale_scheduled_jobs.py

```diff
@@ -0,0 +1,22 @@
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from django.utils import timezone
+
+from nautobot.extras.choices import JobExecutionType
+from nautobot.extras.models import ScheduledJob
+
+
+class Command(BaseCommand):
+    help = "Remove stale scheduled jobs."
+
+    def add_arguments(self, parser):
+        parser.add_argument("max-age", type=int, help="Max age in days")
+
+    def handle(self, *args, **options):
+        ScheduledJob.objects.filter(start_time__lt=timezone.now() - timedelta(days=options["max-age"])).filter(
```

Comment:
Would be nice to have some unit tests for this command - is that something that you can add?